modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordVerifiedConfig.kt

```diff
@@ -0,0 +1,46 @@
+package com.metalbear.mirrord
+
+import com.google.gson.Gson
+import com.intellij.notification.NotificationType
+import com.intellij.util.asSafely
+
+/**
+ * Type for dealing with the output from `mirrord verify-config`.
+ *
+ * It parses the output from the command and displays warnings/errors.
+ *
+ * The command outputs either a `type = "Success"` or `type = "Fail"`.
+ */
+class MirrordVerifiedConfig(private val verified: String, private val notifier: MirrordNotifier) {
+    /**
+     * The errors when the output is `type = "Fail"`.
+     */
+    private val errors: List<String>?
+
+    /**
+     * `verify-config` also outputs the `MirrordConfig` that was set, when `type = "Success"`.
+     */
+    val config: String?
+
+    init {
+        val gson = Gson()
+
+        this.config = gson.fromJson(this.verified, Map::class.java).let { verified ->
```

Comment:
I would aggregate all of the errors and warnings into one notification to avoid spam.

Reply:
honestly doesnt look like spam to me, quite easier to read that way

Reply:
Think the ideal ux for something like this would be an: we display 1 warning/error and have it also say "You have 3 more warnings, click here to see". So we don't bombard the user with a massive wall of text, while also not spamming them with a bunch of stuff.

Guess we merge it this way, then in the future improve on it?

Reply:
i think intellij does that automatically when you have multiple notifications

Reply:
Oh, that's great then!